CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - `streamr resend` is now `streamr stream resend`
   - `streamr generate` is now `streamr mock-data generate`
 - Add storage node commands under `streamr storage-node`
+- Add permission commands: `stream grant-permission` and `stream revoke-permission`
```

Comment:
Needs to be moved under "Unreleased" now (since 5.0.0 was landed).

Reply:
Fixed